services/external-secrets/0.17.0/external-secrets-namespace/external-secrets.yaml

```diff
@@ -0,0 +1,4 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: external-secrets
```

Comment:
Maybe `nkp-external-secrets`?

Reply:
I'll revisit when I get an answer whether this should be user accessible app